apps/src/gamelab/GameLab.js

```diff
@@ -244,6 +264,15 @@ GameLab.prototype.calculateVisualizationScale_ = function () {
 };
 
 /**
+ * @param {string} code The code to search for Data Storage APIs
+ * @return {boolean} True if the code uses any data storage APIs
+ */
+GameLab.prototype.hasDataStoreAPIs = function (code) {
+  return /createRecord/.test(code) || /updateRecord/.test(code) ||
+      /setKeyValue/.test(code);
+};
```

Comment:
I checked with @poorvasingal and she thinks we probably will have the `getKeyValue` and `setKeyValue` - so I went ahead and wrote this so it would work when/if we added them.